static/js/zamboni/devhub.js

```diff
@@ -97,7 +97,7 @@ $(document).ready(function() {
       var $isManualReview = $('#manual-review');
       var $submitAddonProgress = $('.submit-addon-progress');
       function updateListedStatus() {
-        if (!$isUnlisted.is(':checked') || $new_form.data('addon-is-listed')) {  // It's a listed add-on.
+        if (($isUnlisted.length && !$isUnlisted.is(':checked')) || $new_form.data('addon-is-listed')) {  // It's a listed add-on.
```

Comment:
That wouldn't work, because we don't want to assume the add-on is listed if the checkbox isn't there (eg if we're uploading a new version or new file to an existing add-on).